source/triggers/aws-eventbridge.txt

```diff
@@ -152,6 +164,238 @@ Procedure
       rules that trigger on events from that partner event source. For more
       information, see :aws-docs:`Creating a Rule That Triggers on a SaaS Partner Event </eventbridge/latest/userguide/create-rule-partner-events.html>`.
 
+.. _eventbridge_error_handling:
+
+Custom Error Handling
+---------------------
+
+.. note:: Custom Error Handling Does Not Support Authentication and Scheduled Triggers
+
+   Custom error handling supports only database triggers. 
+   Authentication triggers and scheduled triggers are not supported. 
+
+.. figure:: /images/eventbridge-error-handling-diagram.png
+   :alt: EventBridge Custom Error Handling Flow Diagram.
+
+You can create an error function to be executed on a trigger failure, 
+when retry does not succeed. Custom error handling allows you to determine
```

Comment:
how exactly are the number of retries defined for the trigger?

Reply:
will ask Taylor.

Reply:
was mostly curious; it's not vital for these doc updates, I don't think

Reply:
@tcannon91 Anything about retries you think would be helpful to include?

Reply:
I think this is more of an internal thing. We have retries but I am not sure if we want to advertise the number of retries because it is subject to change. Currently it is configured to retry 10 times

Reply:
Will leave out then, thanks!